benchmarks/benchmark_ranges.py

```diff
@@ -89,7 +89,7 @@ def benchmark_union_is_not_none(self, benchmark):
 
     def benchmark_sorting(self, benchmark):
         sorted_ranges_ = []
-        dt = datetime.datetime(2020, 1, 1)
+        dt = datetime.datetime(2020, 1, 1, tzinfo=datetime.timezone.utc)
```

Comment:
What failure does this fix? Could we include that information in the commit message body, please?